tests/SonarPartTwoTest.php

```diff
@@ -7,8 +7,18 @@
 
 class SonarPartTwoTest extends TestCase
 {
-    public function  testGivenOneNumberReturnsZero(): void
+    public function  testGivenOneWindowReturnsSix(): void
     {
         $this->assertSame(6, (new SonarPartTwo())->SweepPartTwo("1\n2\n3"));
     }
+
+    public function  testGivenSecondWindowReturnsNine(): void
+    {
+        $this->assertSame(9, (new SonarPartTwo())->SweepPartTwo("1\n2\n3\n4"));

```

Comment:
are these really the values you are wanting?

Reply:
I've adjusted so the tests return the correct data.